### PR TITLE
arash/footer

### PR DESCRIPTION
**Pull request by arash-fs**

### CODE REVIEW

**Comment by arash-fs** [comment]
@mustofa-binary I think UI/UX change it, in mobile these images should be on below.
Does the delay appear only in this branch? it's may because of `Show` component async request.

File: src/components/layout/footer.js [hunk 1]
```diff
@@ -284,62 +318,56 @@ const Footer = () => (
...
+                        </span>
+                    </LegalIcons>
+                    <LegalTextWrapper>
+                        <TextFooter>
+                            <Localize
+                                text="In the EU, financial products are offered by Binary Investments (Europe) Ltd., W Business Centre, Level 3, Triq Dun Karm, Birkirkara, BKR 9033, Malta, regulated as a Category 3 Investment Services provider by the Malta Financial Services Authority (<0>licence no. IS/70156</0>)."
+                                components={[
+                                    <FooterBoldLink
+                                        key={0}
+                                        target="_black"
```
Comment (mustofa-binary) [inline src/components/layout/footer.js]:
```suggestion
                                        target="_blank"
```

File: src/components/layout/footer.js [hunk 2]
```diff
@@ -284,62 +318,56 @@ const Footer = () => (
...
+                                ]}
+                            />
+                        </TextFooter>
+                        <TextFooter>
+                            <Localize
+                                text="Outside the EU, financial products are offered by Binary (SVG) Ltd, Hinds Building, Kingstown, St. Vincent and the Grenadines; Binary (V) Ltd, Govant Building, Port Vila, PO Box 1276, Vanuatu, regulated by the Vanuatu Financial Services Commission (<0>view licence</0>); Binary (BVI) Ltd, Kingston Chambers, P.O. Box 173, Road Town, Tortola, British Virgin Islands, regulated by the British Virgin Islands Financial Services Commission (<1>licence no. SIBA/L/18/1114</1>); and Binary (FX) Ltd., Lot No. F16, First Floor, Paragon Labuan, Jalan Tun Mustapha, 87000 Labuan, Malaysia, regulated by the Labuan Financial Services Authority to carry on a money-broking business (<2>licence no. MB/18/0024</2>)."
+                                components={[
+                                    <FooterBoldLink
+                                        key={0}
+                                        target="_black"
```
Comment (mustofa-binary) [inline src/components/layout/footer.js]:
```suggestion
                                        target="_blank"
```

File: src/components/layout/footer.js [hunk 3]
```diff
@@ -284,62 +318,56 @@ const Footer = () => (
...
+                                        href="https://www.vfsc.vu/wp-content/uploads/2015/12/List-of-Licensees-under-Dealers-in-Securities-Licensing-Act-CAP-70-18.11.2016.pdf"
+                                    />,
+                                    <FooterBoldLink
+                                        key={1}
+                                        target="_black"
```
Comment (mustofa-binary) [inline src/components/layout/footer.js]:
```suggestion
                                        target="_blank"
```

File: src/components/layout/footer.js [hunk 4]
```diff
@@ -284,62 +318,56 @@ const Footer = () => (
...
+                                        href="/BVI_license.pdf"
+                                    />,
+                                    <FooterBoldLink
+                                        key={2}
+                                        target="_black"
```
Comment (mustofa-binary) [inline src/components/layout/footer.js]:
```suggestion
                                        target="_blank"
```

Comment (arash-fs) [reply src/components/layout/footer.js]: Done

File: src/components/layout/footer.js (hunk 3 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: Done

File: src/components/layout/footer.js (hunk 1 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: Done

File: src/components/layout/footer.js (hunk 2 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: Done

File: src/components/layout/footer.js [hunk 5]
```diff
@@ -183,18 +200,35 @@ const Social = styled.div`
     }
 
     /* TODO: remove this line after having real socal media account */
-    display: none !important;
+    // display: none !important;
+`
+const RiskWarningText = styled.p`
+    line-height: 1.8rem;
+    font-size: 1rem;
```
Comment (mustofa-binary) [inline src/components/layout/footer.js]: this is too small for desktop

File: src/components/layout/footer.js [hunk 6]
```diff
@@ -348,11 +376,11 @@ const Footer = () => (
...
+                            'The financial products offered via this website include binary options, contracts for difference ("CFDs") and other complex derivatives and financial products. Trading binary options may not be suitable for everyone. Trading CFDs carries a high level of risk since leverage can work both to your advantage and disadvantage. As a result, the products offered on this website may not be suitable for all investors because of the risk of losing all of your invested capital. You should never invest money that you cannot afford to lose, and never trade with borrowed money. Before trading in the complex financial products offered, please be sure to understand the risks involved and learn about Responsible Trading.',
```
Comment (mustofa-binary) [inline src/components/layout/footer.js]: `Responsible trading` need to be linked to responsible trading page

Comment (arash-fs) [reply src/components/layout/footer.js]: Done

File: src/components/layout/footer.js (hunk 5 above)
Comment (arash-fs) [reply src/components/layout/footer.js]: Done

**Pull request merged**